M2/Macaulay2/m2/lists.m2

```diff
@@ -175,6 +175,8 @@ rotate(ZZ,VisibleList) := (n,s) -> (
 sort List :=  opts -> internalsort
 rsort List := opts -> internalrsort
 
+List << List := (A, B) -> all(min(#A, #B), i -> A#i <= B#i)
```

Comment:
Let's not add this.  We already have comparison for lists with the operator `<=`, and `<<` doesn't look like a comparison operator.

Reply:
This was discussed in https://github.com/Macaulay2/M2/issues/1945. Quoting your comment there:
> I can't think of a better use for List << List. In the documentation, please compare it to the behavior of List ? List and friends (such as List <= List).

Reply:
Oh, okay, never mind.